build.cake

```diff
@@ -192,9 +203,13 @@ Task("Zip-Files")
     var homebrewFiles = GetFiles( parameters.Paths.Directories.ArtifactsBinFullFx.FullPath + "/**/*");
     Zip(parameters.Paths.Directories.ArtifactsBinFullFx, parameters.Paths.Files.ZipArtifactPathDesktop, homebrewFiles);
 
-    // .NET Core
-    var coreclrFiles = GetFiles( parameters.Paths.Directories.ArtifactsBinNetCore.FullPath + "/**/*");
-    Zip(parameters.Paths.Directories.ArtifactsBinNetCore, parameters.Paths.Files.ZipArtifactPathCoreClr, coreclrFiles);
+    // .NET Core 1.0
+    var coreclrFiles1 = GetFiles( parameters.Paths.Directories.ArtifactsBinNetCore1.FullPath + "/**/*");
+    Zip(parameters.Paths.Directories.ArtifactsBinNetCore1, parameters.Paths.Files.ZipArtifactPathCoreClr, coreclrFiles1);
+    
+    // .NET Core 2.0
+    var coreclrFiles2 = GetFiles( parameters.Paths.Directories.ArtifactsBinNetCore2.FullPath + "/**/*");
+    Zip(parameters.Paths.Directories.ArtifactsBinNetCore2, parameters.Paths.Files.ZipArtifactPathCoreClr, coreclrFiles2);
```

Comment:
I see that the .NET Core 1.0 and 2.0 files are combined in a single zip-file, is that correct?

Reply:
Nope!

Reply:
Okay, I misread then :)

Reply:
Yip zip files should be separate, nuget if possible combined.